src/components/builder/loader.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import {Meta, StoryObj} from '@storybook/react';
+
+import Loader from '@/components/builder/loader';
+
+export default {
+  title: 'Formio/Builder/Loader',
+  component: Loader,
+  parameters: {
+    controls: {hideNoControlsWarning: true},
+    docs: {
+      source: {
+        type: 'dynamic',
+        excludeDecorators: true,
+      },
+    },
+    modal: {noModal: true},
+  },
+} as Meta<typeof Loader>;
```

Comment:
```suggestion
} satisfies Meta<typeof Loader>;
```
this should be preferred since it doesn't just cast the type, it performs the proper type checks still :)

Reply:
Ah okay cool! Yeah that sounds a lot better 👌